src/partials/messenger.ts

```diff
@@ -1031,9 +1041,12 @@ class NavigationController {
     /**
      * Show dialog.
      */
-    public showDialog(name, ev) {
+    public showDialog(name, ev, controller?) {
```

Comment:
You should be able to use default arguments here. Also, maybe add types.

```
public showDialog(name: string, ev: <no-idea>, controller: DialogController = DialogController) {
```

Reply:
The class `DialogController` is not of type `DialogController` :wink:

Reply:
You're missing the point. :wink:

Reply:
Was about to give up on the `controller` type until I found the `Type` hack.